### ISSUE TITLE
Circle packing viz

### ISSUE THREAD

**Issue by harlantwood** [ipfs/dataviz#2]
It would be very easy to add a [circle packing viz like this one](http://mbostock.github.io/d3/talk/20111018/#22) once we are happy with the tree viz in #1 -- the code is nearly identical, just swapping layout.  If you try out the link above, click to zoom.

**Comment by jbenet** [comment]
@harlantwood yes! btw, some of the circle packing ones have nice "exploration" modes. i cant find a good example but imagine http://bl.ocks.org/mbostock/7607535 + a "depth" limit on rendering. when you zoom, more nodes are rendered. good for exploring massive dags.

**Comment by harlantwood** [comment]
Yeah!  Love it.
